app/controllers/api/v1/events_controller.rb

```diff
@@ -1,5 +1,40 @@
-class Api::V1::EventsController < ApplicationController
+class Api::V1::EventsController < ApiController
   def show
     @event = Event.find(params[:id])
   end
+
+  def create
+    @event = Event.new(event_params)
+
+    if @event.save
+      render
+    else
+      render json: {
+        message: 'Validation Failed',
+        errors: @event.errors.full_messages
+      }, status: 422
```

Comment:
We've decided to use the Rails status code symbols instead of the codes themselves -- because sometime people don't remember that 422 is .... an :unprocessable_entity

http://futureshock-ed.com/2011/03/04/http-status-code-symbols-for-rails/